src/Catalog/Registration/RegistrationMakerCatalogItem.cs

```diff
@@ -120,7 +122,7 @@ Uri GetPackageContentAddress()
                 {
                     string id = _catalogItem.GetTriplesWithSubjectPredicate(subject, _catalogItem.CreateUriNode(Schema.Predicates.Id)).FirstOrDefault().Object.ToString().ToLowerInvariant();
                     string version = _catalogItem.GetTriplesWithSubjectPredicate(subject, _catalogItem.CreateUriNode(Schema.Predicates.Version)).FirstOrDefault().Object.ToString().ToLowerInvariant();
-                    string path = string.Format("packages/{0}.{1}.nupkg", id.ToLowerInvariant(), version.ToLowerInvariant());
+                    string path = GetPackagePath(id, version);
```

Comment:
I hate these shorthand functions. I was about to comment "but where's the function?" when I spotted that oneliner above. Any chance we can make it a proper, full-body function? (or am I just whining about how in the old C#5 days things were better ;-))